### ISSUE TITLE
[HL1S] [Steampipe] Map c1a2 Bug 64

### ISSUE THREAD

**Issue by Pinhead-alejandro** [Theuaredead/Half-Life-Source-Update#67]
This fan, is terrible, I saw it stoped, I saw it working to be stoped happendly, and today I saw it working perfectly. But it has difficults... this is clear to me:

![2015-07-14_00018](https://cloud.githubusercontent.com/assets/13278870/8680221/2b743780-2a60-11e5-82ff-08d014dc59e9.jpg)

**Comment by Theuaredead** [comment]
Yeah, I've noticed this fan being weird, when we were testing out the code on Source Base 2007 (since that's what the code we received was made for, and we wanted to know if the game was in working order) the fan wasn't working at all, but when we converted to 2013 it started working again.


So if this is map related or code related, beats the hell out of me

**Comment by Pinhead-alejandro** [comment]
Well, be sure that I will indicate along the titles any difference when HD pack is desactivated... maybe, this fan will work with this HD pack desactivated. We will see it. But first, I need end all my reports. XDD
